docs/csharp/language-reference/toc.yml

```diff
@@ -1,431 +1,429 @@
 items:
 - name: C# language reference
+  href: ./index.yml
+- name: Configure language version
+  href: ./configure-language-version.md
+- name: Types
   items:
-  - name: Overview
-    displayName: language reference
```

Comment:
The other 2 TOCs have only one "root" node, with an "items node that have every other item under it. It would look something like this here. (Not a suggestion, because I can't create a suggestion on deleted lines:

```yml
items:
- name C# language reference
  href: ./index.yml
  items:
  -name: Configure language version
    href: ./configure-language-version.md
  -name: Types
    items:
    # remaining items
```